cpp/lib/src/DbConnection.cc

```diff
@@ -436,6 +436,26 @@ std::string DbConnection::escapeString(const std::string &unescaped_string, cons
 }
 
 
+// Creates two hex digits for each character in "s".
+static std::string StringToHexString(const std::string &s) {
```

Comment:
Move to StringUtil?

Reply:
I don't know if this is generic enough.  I can't think of a typical application.